app/views/histories/10_downing_street.html.erb

```diff
@@ -1,17 +1,9 @@
+<% content_for :header do %>
```

Comment:
This change causes a visual difference, because after moving breadcrumbs out of main, they are in different parts of the page. In the previous version the bottom margin of breadcrumbs collapsed with the top margin of the title. But as those are now separated by many levels of element structure, some elements of which do things like `clear`, the margins cannot collapse anymore.

But this makes it more consistent with the component system, as the margins are now the default margins of the components. 

If we want to stick to the old look, I'd need to apply a custom CSS instead of using component defaults, which defeats the purpose of having the component system.

Reply:
EDIT:

I've added a separate commit which mitigates the visual changes without creating a custom CSS. Now the changes are only visible on the mobile and the space is slightly smaller, so it takes less space on the mobile screen, and it also keeps the current look without any visual changes on the desktop.

Reply:
Good spot on the visual difference, this has highlighted something that we can improve across all templates in `government-frontend` and likely other frontend applications as well.

Looking at the guidance from the Design System, we should add the missing `govuk-main-wrapper` class to the `main` element - https://design-system.service.gov.uk/styles/layout/#add-vertical-space

`whitehall` has a good example which conditionally loads the `govuk-main-wrapper--l` if there are no breadcumbs or back-link component - https://github.com/alphagov/whitehall/blob/e6bdcf39d1a91a0552c5327ccbb42b097aaaa19d/app/views/layouts/design_system.html.erb#L33

This would fix the visual changes mentioned here and avoid the need to add margin-top to the title, but it could cause issues in other templates, so further testing may be required, this may be best done as a separate pull request, I will create a Trello card for this detailing the issue and the fix.

Reply:
Do you think anything else needs to be done in this PR, or it is fine and the changes you mentioned would go to the PR you mentioned?

Reply:
I discussed with @MartinJJones and he had a nice idea how to fix this, which I have now applied, so the problem is now resolved.

Reply:
Nice work, thanks for spotting the issue and fixing 👍